### PR TITLE
Upgrade to .NET Core SDK version 2.1.401

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by JoeRobich** [comment]
test WIP

File: docs/contributing/Building, Debugging, and Testing on Windows.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ If a stack trace is displayed on .NET Framework older than 4.7.2 (e.g. by xUnit
 1. [Visual Studio 2017 Version 15.7](https://www.visualstudio.com/vs/preview/)
     - Ensure C#, VB, MSBuild, .NET Core and Visual Studio Extensibility are included in the selected work loads
     - Ensure Visual Studio is on Version "15.7" or greater
-1. [.NET Core SDK 2.1.300](https://www.microsoft.com/net/download/core) (the current previews are: [Windows x64 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300-rtm-008866/dotnet-sdk-2.1.300-rtm-008866-win-x64.exe), [Windows x86 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300-rtm-008866/dotnet-sdk-2.1.300-rtm-008866-win-x86.exe))
+1. [.NET Core SDK 2.1.401](https://www.microsoft.com/net/download/core) (the current previews are: [Windows x64 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.401/dotnet-sdk-2.1.401-win-x64.exe), [Windows x86 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.401/dotnet-sdk-2.1.401-win-x86.exe))
```
Comment (jasonmalinowski) [inline docs/contributing/Building, Debugging, and Testing on Windows.md]: Does this need to say "current previews" anymore?

**Comment by jaredpar** [comment]
retest windows_release_unit32_prtest please

**Comment by jaredpar** [comment]
Added justificatino for retry into the bug. https://github.com/dotnet/roslyn/issues/29779

**Pull request merged**